juvix-mode/juvix-mode.el

```diff
@@ -69,7 +69,7 @@
         )
     (with-temp-buffer
       (let ((
-             cmd-str (concat "juvix " (if juvix-disable-embedded-stdlib "--no-stdlib " "") "dev scope "
+             cmd-str (concat "juvix " (if juvix-disable-embedded-stdlib "--no-stdlib " "") "dev scope --with-comments "
```

Comment:
Please make this change in https://github.com/anoma/juvix-mode, and remove it from this PR.

Reply:
I'll do it after this gets merged